components/dsl/resources/ome/dsl/object.vm

```diff
@@ -319,6 +319,15 @@ implements java.io.Serializable, IObject
     */
     private static final long serialVersionUID = 0000000030000020302L;
 
+#if($type.isEnum && !$type.shortname.startsWith("Units") && ${type.shortname} != "Binning" && ${type.shortname} != "PixelsType")
```

Comment:
I think we should expose the pixels type too since we have static values in various classes e.g.
``PlaneFactory``.
Maybe in that case drop the ``VALUE``

Also I feel that the prefix should be upper case to match the usual way of introducing static fields and leads to a mix of upper/lower cases constants e.g.
``
public final static String VALUE_Brightfield = "Brightfield";
    public final static String VALUE_Phase = "Phase";
    public final static String VALUE_DIC = "DIC";
    public final static String VALUE_HoffmanModulation = "HoffmanModulation";
    public final static String VALUE_ObliqueIllumination = "ObliqueIllumination";
    public final static String VALUE_PolarizedLight = "PolarizedLight";
    public final static String VALUE_Darkfield = "Darkfield";
    public final static String VALUE_Fluorescence = "Fluorescence";
    public final static String VALUE_Other = "Other";
    public final static String VALUE_Unknown = "Unknown";
``